server/src/infra/entities/system-config.entity.ts

```diff
@@ -105,6 +107,7 @@ export enum SystemConfigKey {
 export enum TranscodePolicy {
   ALL = 'all',
   OPTIMAL = 'optimal',
+  BY_BITRATE = 'by_bitrate',
```

Comment:
The value should be in camelCase. Could you make this `byBitrate` instead? Actually, you could just name this enum `BITRATE`.

Reply:
I wasn't sure, how those values will be interpreted/parsed by open-api etc, so I went the safe way here and just made it lowercase as the others. By I will change it today, thank you.

Reply:
Done

Reply:
@mertalev Are there still any problems with this PR? 
I ask because the other PR was merged very quickly, while here is no activity even if the changes are simpler. 😅

Reply:
Sorry! I think you should rebase on main since the diff shows the accepted codec stuff that's already merged.

Reply:
You should also add unit tests for this.